examples/settings.py

```diff
@@ -95,6 +98,23 @@ def make_cfg(settings):
         ortho_sensor_spec.sensor_subtype = habitat_sim.SensorSubType.ORTHOGRAPHIC
         sensor_specs.append(ortho_sensor_spec)
 
+    if settings["fisheye_sensor"]:
+        fisheye_sensor_spec = habitat_sim.FisheyeSensorDoubleSphereSpec()
+        fisheye_sensor_spec.uuid = "fisheye_sensor"
```

Comment:
I would change the uuid as well.

Reply:
Seriously, you can change it to `fisheye_color_sensor`. You do not have to wait until you refactor it in the future. It must be corrected anyway, sooner or later.

Reply:
Oh, I see. you are passing the parameters in.